packages/apps-config/src/api/spec/subspace.ts

```diff
@@ -35,9 +36,14 @@ function extractAuthor (
   const preRuntimes = digest.logs.filter(
     (log) => log.isPreRuntime && log.asPreRuntime[0].toString() === 'SUB_'
   );
+
+  if (!preRuntimes || preRuntimes.length === 0) {
```

Comment:
`preRuntimes.length === 0` can be replaced with `!preRuntimes.length `

Reply:
done